### ISSUE TITLE
Paper Typo

### ISSUE THREAD

**Issue by zaptrem** [Lakonik/GMFlow#1]
The paper mentions guidance_scales like 0.4, but the config has 0.04 https://github.com/Lakonik/GMFlow/blob/51c41ef15692cb3ca0b3a1375cd56eb4f25ca441/configs/gmflow_imagenet_k8_8gpus.py#L96

Is this a typo in the paper?

**Comment by Lakonik** [comment]
Thank you for the question! 0.4 is the scale for best Precision, whereas 0.04 is the scale for best FID. In the training script we use 0.04 for the validation step. A full sweep through the scales is used in the testing script for comprehensive evaluation:
https://github.com/Lakonik/GMFlow/blob/51c41ef15692cb3ca0b3a1375cd56eb4f25ca441/configs/gmflow_imagenet_k8_test.py#L58

**Comment by zaptrem** [comment]
> Thank you for the question! 0.4 is the scale for best Precision, whereas 0.04 is the scale for best FID. In the training script we use 0.04 for the validation step. A full sweep through the scales is used in the testing script for comprehensive evaluation:
> 
> [GMFlow/configs/gmflow_imagenet_k8_test.py](https://github.com/Lakonik/GMFlow/blob/51c41ef15692cb3ca0b3a1375cd56eb4f25ca441/configs/gmflow_imagenet_k8_test.py#L58)
> 
> Line 58 in [51c41ef](/Lakonik/GMFlow/commit/51c41ef15692cb3ca0b3a1375cd56eb4f25ca441)
> 
>  guidance_scales = [0.02, 0.03, 0.04, 0.05, 0.06, 0.07, 0.08, 0.09, 0.11, 0.13, 0.16, 0.19, 0.23, 0.27, 0.33, 0.39, 0.47, 0.55, 0.65, 0.75]

Thanks! When you cranked up the guidance scale to 0.75 or even 0.9 did you see saturation return? I have an interesting usecase where with normal diffusion the proper CFG (the one that avoids oversaturation while delivering coherent results) varies significantly from sample to sample and am hoping this approach can finally offer a direct trade off between diversity and fidelity without oversaturation issues (similar to how CFG on logits in token space works). However, when I reimplemented your technique I'm still seeing oversaturation for certain values, though there's a very good chance my version has bugs I have yet to find.

**Comment by Lakonik** [comment]
A very large guidance scale (e.g., w>0.7) would still cause oversaturation, since the guidance bias may accumulate along the same direction during the sampling iterations. 

Still, if your implementation is correct (which requires a flow matching model with at least one Gaussian component plus learned variance), then probabilistic guidance should significantly reduce oversaturation compared to standard CFG.
